enclave-tls/src/lib/tls_wrappers/wolfssl/cleanup.c

```diff
@@ -0,0 +1,24 @@
+#include <enclave-tls/log.h>
+#include <enclave-tls/tls_wrapper.h>
+
+#include "wolfssl_private.h"
+
+/* *INDENT-OFF* */
+tls_wrapper_err_t wolfssl_cleanup(tls_wrapper_ctx_t *ctx)
+{
+	ETLS_DEBUG("tls_wrapper_wolfssl cleanup() called\n");
+
+	wolfssl_ctx_t *ws_ctx = (wolfssl_ctx_t *)ctx->tls_private;
+	if (ws_ctx->ssl != NULL) {
```

Comment:
The check for ws_ctx should be executed at first. And if ws_ctx is NULL, it is necessary return an error code.